static/app/utils/performance/vitals/constants.tsx

```diff
@@ -116,4 +116,52 @@ export const MOBILE_VITAL_DETAILS: Record<MobileVital, Vital> = {
     ),
     type: measurementType(MobileVital.FramesFrozen),
   },
+  [MobileVital.FramesSlowRate]: {
+    slug: 'frames_slow_rate',
+    name: t('Slow Frames Rate'),
+    description: t(
+      'Slow Frames Rate is the percentage of frames recorded within a transaction that is considered slow.'
+    ),
+    type: measurementType(MobileVital.FramesSlowRate),
+  },
+  [MobileVital.FramesFrozenRate]: {
+    slug: 'frames_frozen_rate',
+    name: t('Frozen Frames Rate'),
+    description: t(
+      'Frozen Frames Rate is the percentage of frames recorded within a transaction that is considered frozen.'
+    ),
+    type: measurementType(MobileVital.FramesFrozenRate),
+  },
+  [MobileVital.StallCount]: {
+    slug: 'stall_count',
+    name: t('Stalls'),
+    description: t(
+      'Stalls is the number of times the application stalled within a transaction.'
+    ),
+    type: measurementType(MobileVital.StallCount),
+  },
+  [MobileVital.StallTotalTime]: {
+    slug: 'stall_total_time',
+    name: t('Stall Total Time'),
+    description: t(
+      'Stall Total Time is the total amount of time the application is stalled within a transaction.'
+    ),
+    type: measurementType(MobileVital.StallTotalTime),
+  },
+  [MobileVital.StallLongestTime]: {
+    slug: 'stall_longest_time',
+    name: t('Stall Longest Time'),
+    description: t(
+      'Stall Longest Time is the longest amount of time the application is stalled within a transaction.'
+    ),
+    type: measurementType(MobileVital.StallLongestTime),
+  },
+  [MobileVital.StallRate]: {
+    slug: 'stall_rate',
+    name: t('Stall Rate'),
```

Comment:
I'm not sure stall time / transaction.duration is actually a `rate`.

It's more of a stalled_percentage maybe?

Reply:
will update to `stalled_percentage` 👍